.github/workflows/cas-build.yml

```diff
@@ -100,6 +100,10 @@ jobs:
             ${{ runner.os }}-gradle-
       - name: Build with Gradle
         run: ./gradlew --build-cache --configure-on-demand --no-daemon build -x test -x javadoc -x check --parallel
+      - name: Run tests on Windows and Mac
+        run: ./testcas.sh --category simple,mfa,files,groovy,webflow,webflow-config,webflow-actions,webflow-events,webflow-mfa-actions,attributes,services,expiration-policy
```

Comment:
Note that this cannot pass. You want the same set of tests to run all platforms equally, without any manual work whatsoever. Maintaining a separate list of test categories for windows vs unix vs something else is never going to be maintainable long-term.